composer.json

```diff
@@ -31,11 +31,11 @@
   "require-dev": {
     "phpunit/phpunit": "4.8.*",
     "squizlabs/php_codesniffer": "2.*",
-    "phpdocumentor/phpdocumentor": "2.*"
+    "phpdocumentor/phpdocumentor": "2.8.*"
   },
   "autoload": {
     "psr-4": {
-      "Google\\Cloud\\": "src"
+      "Gcloud\\": "src"
```

Comment:
You're right, the other gcloud libraries (with the exception of [go](https://github.com/GoogleCloudPlatform/gcloud-golang/blob/master/cloud.go#L17)) use `gcloud` in the namespace/package name.

However, @tmatsuo pointed out [psr-4](http://www.php-fig.org/psr/psr-4/#2-specification) states the top-level namespace should be a "vendor namespace", I think we should use `Google\` there.

So, `Google\Gcloud` works for me.